### PR TITLE
Add Recite mode prompt

### PR DESCRIPTION
**Pull request by calebwoods**
Also did some cleanup of some unneed functions

In action:

recite_mode

### CODE REVIEW

File: css/components/_passage.css [hunk 1]
```diff
@@ -120,3 +120,34 @@
...
+
+.reciteMode {
+  text-align: center;
+
+  .actions {
+    display: flex;
+    justify-content: space-around;;
+
+    button {
+      background-color: $vintage-eggy;
```
Comment (defnotryan) [inline css/components/_passage.css:132]: I feel like switching up the colors for these buttons might be a good idea? Consider `$vintage-slimer`?

File: css/components/_passage.css [hunk 2]
```diff
@@ -120,3 +120,34 @@
...
+      flex-grow: 1;
+      padding: 0.25em;
+      margin: 0 0.25em;
+      text-align: center;
+      text-transform: uppercase;
+      user-select: none;
+    }
+
+    .restart {
+      background: $vintage-background;
```
Comment (defnotryan) [inline css/components/_passage.css:148]: See above -- consider `$vintage-rust` with `$vintage-white` text?

**Comment by defnotryan** [comment]
Just some thoughts about colors; we may or may not want to change them, idk. Other than that looks good!

**Comment by calebwoods** [comment]
Here are some the experiments I did with colors.  Overall seemed to clash, seemed like keeping the same button color added for cohesion to the app.

Punchy and Slimer:

screen shot 2016-03-05 at 11 34 56 am

Rust and Slimer:

screen shot 2016-03-05 at 11 36 49 am

Gray and Slimer:

screen shot 2016-03-05 at 11 37 23 am

**Comment by defnotryan** [comment]
@calebwoods Hm yeah I see what you mean. I guess let's leave it as white and eggy unless anyone else has a better idea.

**Pull request merged**